docs/api.yml

```diff
@@ -0,0 +1,189 @@
+openapi: "3.0.0"
+info:
+  title: RT Dashboard API
+  version: 1.0.0
+
+paths:
+#  /:
+#  /{queue_name}:
+#  /{queue_name}/{page}:
+#  /history:
+  /history.json:
+    get:
+      summary: List history entries
+      responses:
+        200:
+          description: |
+            Return object containting list of rows and groups of history entries
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  rows:
+                    type: array
+                  groups:
+                    type: array
```

Comment:
@groupsky i did not further specify the type/shapes here, but could easily be added/extended if needed